AmplifyPlugins/DataStore/Tests/AWSDataStorePluginTests/Sync/RemoteSyncEngineTests.swift

```diff
@@ -46,16 +46,21 @@ class RemoteSyncEngineTests: XCTestCase {
     }
 
     func testErrorOnNilStorageAdapter() throws {
+        guard let remoteSyncEngine = remoteSyncEngine else {
+            XCTFail("Failed to initialize remoteSyncEngine")
+            return
+        }
+
         let failureOnStorageAdapter = expectation(description: "Expect receiveCompletion on storageAdapterFailure")
 
         storageAdapter = nil
         let remoteSyncEngineSink = remoteSyncEngine
             .publisher
             .sink(receiveCompletion: { _ in
                 failureOnStorageAdapter.fulfill()
-        }, receiveValue: { _ in
-            XCTFail("We should not expect the sync engine not to continue")
-        })
+            }, receiveValue: { _ in
+                XCTFail("We should not expect the sync engine not to continue")
```

Comment:
thanks for providing the deep link to the circleCI failure in the PR, I noticed this is the message in the logs, so does it receive a value in the subscription when running on macOS?